README.md

```diff
@@ -149,10 +149,10 @@ source env/pulpissimo.sh
 To build the RTL simulation platform, start by getting the latest version of the
 IPs composing the PULP system:
 ```bash
-./update-ips
+make checkout
 ```
 This will download all the required IPs, solve dependencies and generate the
-scripts by calling `./generate-scripts`.
+scripts. The default dependency management tool is IPApproX, where `./update-ips` and `./generate-scripts` are called. If the environment variable `BENDER` is set, bender is used as the dependency management tool.
```

Comment:
We should ideally have a single set of commands, that only discriminates between the usage of the BENDER env variable. I.e. we should have a `checkout` `scripts` `build` target that does the job for both tools. Right now, generate scripts is done with a make target for bender but with a Python script for IPApprox. That's a bit inconsistent.

Reply:
This makes sense, I have updated it, although this is likely to cause confusion for IPApproX users, as the behavior of `./update-ips` is now different from before, requiring an additional step.